baker/postUpdatedHook.ts

```diff
@@ -17,12 +17,24 @@ const zeroDateString = "0000-00-00 00:00:00"
 const syncPostToGrapher = async (
     postId: number
 ): Promise<string | undefined> => {
-    const rows = await wpdb
-        .getSingleton()
-        .query(
-            "SELECT * FROM wp_posts WHERE ID = ? AND post_status != 'trash'",
-            [postId]
-        )
+    const rows = await wpdb.getSingleton().query(
+        `--sql
+        -- we select all the fields from wp_posts and then we also
+        -- json array aggregate the authors. The authors come as strings like
+        -- Charlie Giattino Charlie Giattino Charlie Giattino 44 charlie@ourworldindata.org
+        -- so we use regexp_replace to cut out the first two words
+        SELECT p.*, JSON_ARRAYAGG(
+            JSON_OBJECT('author',
+                regexp_replace(t.description, '^([[:alnum:]]+) ([[:alnum:]]+) .+$' , '$1 $2'),
```

Comment:
Okay, I hate to break this to you, but: This doesn't work for `Esteban Ortiz-Ospina` (see https://regexr.com/782mi).
It also won't work as intended for double names.

Reply:
Oh and to clarify - this regex is still not perfect and as you say, double names without dashes are still a problem. But I compared this with the authors we actually have so far and this seems to capture all of the correctly. The alternative would be to split the tokens at whitespaces and work out the structure but this one seems more fitting for this particular case